app/src/main/java/io/islnd/android/islnd/app/database/IslndDb.java

```diff
@@ -268,4 +270,33 @@ public static void deleteComment(
         // Delete from network
         Rest.postComment(encryptedComment, Util.getApiKey(context));
     }
+
+    public static void syncServerTime(Context context, boolean force) {
+        final String prefKey = context.getString(R.string.server_time_offset);
+
+        SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
We might want to have knowledge of this shared preference inside the ServerTime class. Currently, the code outside of that class knows that the ServerTime class is using an offset and is deciding to set it or not. Instead, we could have the ServerTime class hide that implementation detail and provide a consistent interface of the server time without exposing an offset field to the outside. Essentially, we can think of this offset as a private variable that only the ServerTime should know about.